README.md

```diff
@@ -24,7 +26,21 @@ or, if you want to have the website automatically update as you make changes:
 bundle exec jekyll serve --watch
 ```
 
-Contents:
+## Build with docker
+
+If you have docker installed, you don't need to install anything else to build the website and view it.
+
+Run:
+
+```bash
+docker run -p 4000:4000 -v $(pwd):/site bretfisher/jekyll-serve
+```
+
+And then go to http://0.0.0.0:4000 to view the website.
+It will create the website in the  `_site` folder and it will be owned by root.
+To remove the folder run `sudo rm -fr _site/`.
```

Comment:
this worked for me, but it was not owned by root and I was able to remove the folder. Is this os-dependent? And I'd prefer not to have `sudo` in instructions unless absolutely necessary.

Reply:
Yes it will work different for every OS since they all do docker a bit differently, I think we should change it to something like "Hey depending on how docker works on your platform, the `_site` folder may be owned by root instead of your user, you can use sudo to remove it if needed"

Reply:
yes that's perfect! just avoiding people naively copy/pasting a `sudo rm` command.

Reply:
```suggestion
Depending on your OS, the `_site` directory may be owned by root (instead of your user), and you will need to use `sudo` to remove it.

```